# Each e-stop name in e_stop_topics needs to be in e_stop_list!

e_stop_list:
  - name: "hard_remote_e_stop"
    value: false
  - name: "soft_remote_e_stop"
    value: false
  - name: "big_red_button_e_stop"
    value: false
  - name: "ui_e_stop"
    value: false

e_stop_topics:
  - name: "/emergency_stop_hardware"
    e_stops:
      - "big_red_button_e_stop"
      - "hard_remote_e_stop"
  - name: "/emergency_stop_software"
    e_stops:
      - "soft_remote_e_stop"
      - "ui_e_stop"
